### PR TITLE
Added Rule 5

### PR DESCRIPTION
**Pull request by soarn**
If your usermode contains +B you will be kicked! Do not pretend to be a bot!

### CODE REVIEW

**Review by Chew — COMMENTED** [review]
This seems channel specific. "kicked" implies from a channel. Reminder these are global network rules.

File: info/rules.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ By joining the server, you agree to following these very simple rules.
 
 4) Spamming (or pinging) any network staff for any reason is strictly prohibited unless they specifically allow it in a channel.
 
-5) If your usermode includes +B you will be kicked! Do not pretend to be a bot!
+5) If your usermode includes +B you will get a network wide kick (kill)
```
Comment (Chew) [inline info/rules.md]: You could probably do "be killed (network-wide kick)"

**Comment by Chew** [comment]
Although I wouldn't moderate this or punish for it. What exactly is your reasoning for adding this?

Mode +B has one use: Marking the user as a bot. What harm does a user _95% of the time jokingly_ cause by enabling +B?

**Comment by soarn** [comment]
I added +B to my umode and you kicked me (jokingly) but still, also i'll change the rule to what you suggested when I get on a computer.

**Comment by Chew** [comment]
Did we forget about this?

**Comment by soarn** [comment]
yeah, so to merge or not to merge, that is the question...

**Comment by Chew** [comment]
In case you forgot, I'm waiting for you to actually do the changes I requested.

**Comment by soarn** [comment]
Changes committed, waiting for
>M
>E
>R
>G
>E

**Comment by soarn** [comment]
BE GONE BOT

**Comment by Chew** [comment]
Sorry stalebot! I hate when people leave a comment to unstale the pr. feel free to close whenever!

**Pull request closed without merging**